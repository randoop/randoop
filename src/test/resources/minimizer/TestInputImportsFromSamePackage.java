import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputImportsFromSamePackage {

  /* This test input uses ClassA belonging to package dir_a and ClassB belonging
  to package dir_a. The minimizer should simplify the fully-qualified type name
  for instances of either types.
   */

  @Test
  public void test1() throws Throwable {
    test.minimizer.dir_a.ClassA aObject = new test.minimizer.dir_a.ClassA();
    test.minimizer.dir_a.ClassB bObject = new test.minimizer.dir_a.ClassB();

    aObject.setId(1);
    bObject.setId(2);

    org.junit.Assert.assertFalse(aObject.getId() == bObject.getId());

    aObject.setId(100);
    bObject.setId(100);
    // Fails, should be true
    org.junit.Assert.assertFalse(aObject.getId() == bObject.getId());
  }
}
