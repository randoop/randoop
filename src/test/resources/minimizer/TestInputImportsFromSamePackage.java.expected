import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;
import test.minimizer.dir_a.ClassA;
import test.minimizer.dir_a.ClassB;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputImportsFromSamePackageMinimized {

    /* This test input uses ClassA belonging to package dir_a and ClassB belonging
  to package dir_a. The minimizer should simplify the fully-qualified type name
  for instances of either types.
   */

    @Test
    public void test1() throws Throwable {
        ClassA aObject = new ClassA();
        ClassB bObject = new ClassB();
        aObject.setId(100);
        bObject.setId(100);
        // Fails, should be true
        org.junit.Assert.assertFalse(aObject.getId() == bObject.getId());
    }
}
